samples/azure-quantum/simulators/SparseSimulator.ipynb

```diff
@@ -0,0 +1,2274 @@
+{
```

Comment:
Please make sure to make this notebook available in other samples experiences — the portal experience is only one of our four samples experiences, such that this notebook must be added to `main` as well before the sample can be considered complete. Thanks!

Reply:
DevOps item 39303.
I assume the folder in `main` will be the same as [in AQ gallery branch](https://github.com/microsoft/Quantum/pull/637#discussion_r839937259). Otherwise please specify the folder in `main`.

Reply:
https://github.com/microsoft/Quantum/issues/639
I assume that the folder in `main` will be the same as [in the AQ gallery branch](https://github.com/microsoft/Quantum/pull/637#discussion_r839937259), otherwise please specify the folder in `main`.

Reply:
Please see comment above. Thanks!